Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -61,9 +61,8 @@ public class NavigatorIndex {
     
     /// A specific error to describe issues when processing a `NavigatorIndex`.
     public enum Error: Swift.Error, DescribedError {
-        
         /// Missing bundle identifier.
-        case missingBundleIndentifier
+        case missingBundleIdentifier
```

Comment:
Same question that I had [here][1]:

Since this is a public enum case it's technically a source breaking change
```suggestion
        case missingBundleIdentifier
        
        @available(*, deprecated, renamed: "missingBundleIdentifier")
        case missingBundleIndentifier
```

[1]: https://github.com/apple/swift-docc/pull/645/files#r1242520665